src/app/helpers/backbone/view.js

```diff
@@ -80,6 +79,70 @@ class BaseView extends Backbone.View {
         this.regions = new Regions(this.regions, this);
     }
 
+    delegate(eventName, selector, listener) {
```

Comment:
It's overriding a few Backbone methods (that are also used by Backbone.NativeView).  Basically I think the event delegation is slightly broken (which causes events to fail to attach correctly in some instances... you may not have run into it yet).

I'm planning on opening a PR for Backbone.NativeView as well to see if I can get this merged upstream, but that might take some time.